### PR TITLE
base-town.yaml: Add more guild leaders for levelup to use.

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

File: data/base-town.yaml [hunk 1]
```diff
@@ -82,16 +82,16 @@ Crossing:
   almsbox:
     id: 741
   guild_leaders:
-    Bard: 7892
+    Bard: 7879
```
Comment (rpherbig) [inline data/base-town.yaml]: Nice catch

File: data/base-town.yaml [hunk 2]
```diff
@@ -199,6 +199,14 @@ Shard:
     inner_room:
   almsbox:
     id: 13143
+  guild_leaders:
```
Comment (rpherbig) [inline data/base-town.yaml]: Would you be up for including the entire list for each city (similar to https://github.com/rpherbig/dr-scripts/commit/d82da1536095c75fb23c1428896efbb516eab40e#diff-e65384bcbc952bdd340d0bf9c06b452cR84) so we can see at a glance which ones are missing?

Comment (Sarvatt) [reply data/base-town.yaml]: Done, also made leth use crossing guildleaders and boar clan use the same as hib ones.

File: data/base-town.yaml [hunk 3]
```diff
@@ -265,7 +279,18 @@ Riverhaven:
   stamina:
     outer_room: 578
     inner_room:
-  # repairs: 8737
```
Comment (Sarvatt) [inline data/base-town.yaml]: deleted this comment because repairs are already defined

**Pull request merged**